sphinxcontrib/mermaid.py

```diff
@@ -281,6 +281,29 @@ def render_mm_latex(self, node, code, options, prefix='mermaid'):
         self.builder.warn('mm code %r: ' % code + str(exc))
         raise nodes.SkipNode
 
+    if self.builder.config.mermaid_pdfcrop != '':
+        mm_args = [self.builder.config.mermaid_pdfcrop, outfn]
+        try:
+            p = Popen(mm_args, stdout=PIPE, stdin=PIPE, stderr=PIPE)
+        except OSError as err:
+            if err.errno != ENOENT:   # No such file or directory
+                raise
+            self.builder.warn('command %r cannot be run (needed to crop pdf), check the mermaid_cmd setting' % self.builder.config.mermaid_pdfcrop)
+            return None, None
+
+        stdout, stderr = p.communicate()
+        if self.builder.config.mermaid_verbose:
+            self.builder.info(stdout)
+
+        if p.returncode != 0:
+            raise MermaidError('PdfCrop exited with error:\n[stderr]\n%s\n'
+                                '[stdout]\n%s' % (stderr, stdout))
+        if not os.path.isfile(outfn):
+            raise MermaidError('PdfCrop did not produce an output file:\n[stderr]\n%s\n'
+                                '[stdout]\n%s' % (stderr, stdout))
+
+        fname = os.path.splitext(fname)[0] + "-crop" + os.path.splitext(fname)[1]
```

Comment:
could you use `format()` here ? so there is no need to call `os.path.splitext(fname)` twice

Reply:
yes, will update the patch tomorrow!

BR bastian